src/main/java/io/phasetwo/service/auth/OrgAddUserAuthenticatorFactory.java

```diff
@@ -14,8 +13,13 @@
 import org.keycloak.models.KeycloakSession;
 import org.keycloak.models.KeycloakSessionFactory;
 import org.keycloak.models.RealmModel;
+import org.keycloak.models.UserModel;
 import org.keycloak.provider.ProviderEvent;
 
+import java.util.Map;
+
+import static io.phasetwo.service.Orgs.ORG_OWNER_CONFIG_KEY;
+
```

Comment:
can you make sure to run `mvn fmt:format` before committing? I think that will make the imports consistent.